vault/token_store.go

```diff
@@ -774,6 +774,9 @@ func (ts *TokenStore) create(ctx context.Context, entry *logical.TokenEntry) err
 	}
 
 	entry.Policies = policyutil.SanitizePolicies(entry.Policies, policyutil.DoNotAddDefaultPolicy)
+	if len(entry.Policies) == 1 && entry.Policies[0] == "root" {
+		metrics.IncrCounter([]string{"token", "createRoot"}, 1)
```

Comment:
I don't know if most of our metrics are `create-root` or `create_root` but I'm pretty sure they're not this kind of case.

Reply:
Sorted, thanks